packages/core/src/salt-provider/SaltProvider.tsx

```diff
@@ -262,6 +277,7 @@ function InternalSaltProvider({
   const density = densityProp ?? inheritedDensity ?? DEFAULT_DENSITY;
   const themeName =
     themeProp ?? (inheritedTheme === "" ? DEFAULT_THEME_NAME : inheritedTheme);
+  const brandName = brandProp ?? inheritedBrand ?? DEFAULT_BRAND;
   const mode = modeProp ?? inheritedMode;
   const breakpoints = breakpointsProp ?? DEFAULT_BREAKPOINTS;
   const corner = cornerProp ?? inheritedCorner ?? DEFAULT_CORNER;
```

Comment:
(I haven't put too much thought into this.) Will styling option inheritance cause any issue, when nest provider between brand and legacy?

Reply:
thanks, will investigate this